.controlplane/entrypoint.sh

```diff
@@ -18,8 +18,11 @@ echo " -- Waiting for services"
 wait_for_service $(echo $DATABASE_URL | sed -e 's|^.*@||' -e 's|/.*$||')
 wait_for_service $(echo $REDIS_URL | sed -e 's|redis://||' -e 's|/.*$||')
 
-echo " -- Preparing database"
-rails db:prepare
+# If running the rails server then create or migrate existing database
+if [ "${1}" == "./bin/rails" ] && [ "${2}" == "server" ]; then
+  echo " -- Preparing database"
+  ./bin/rails db:prepare
+fi
```

Comment:
@dzirtusss should this be recommended?

See https://chat.openai.com/share/f7e6d78b-ab5c-4e2f-808c-12c717053613

Reply:
@ahangarha why is this duplicated in the bin script in this PR?

Reply:
The `bin/docker-entrypoint` file comes from new Rails setup.